### PR TITLE
enhance Eduroam configuration (PWD, error messages)

### PR DESCRIPTION
**Pull request by jacqueline8711**
## Issue

This fixes the following issue(s): #530 #531 #532

## Screenshot
screenshot_20171201-155940
screenshot_20171201-160000
screenshot_20171201-160019

### CODE REVIEW

**Comment by jacqueline8711** [comment]
Can somebody check if the PWD authentication works for SDK < 18 ? I don't have a phone with such an old operating system and the emulator doesn't seem to support Wifi

**Comment by jacqueline8711** [comment]
I haven't found a way to check if the profile wasn't created by the TCA in advance.
Well there is an attribute in the WifiConfiguration class that seems to show exactly what we want to know but it seems like it isn't accessible to a non-system application

For example here it shows that eduroam was configured by the eduroamCAT app:
screen shot 2017-12-01 at 13 12 54

File: app/src/main/java/de/tum/in/tumcampusapp/activities/SetupEduroamActivity.java [hunk 1]
```diff
@@ -33,6 +34,8 @@
     private EditText lrz;
     private EditText password;
 
+    public static final String EXTRA_FOREIGN_CONFIGURATION_EXISTS = "CONFIGURED_BY_OTHER_APP";
```
Comment (kordianbruck) [inline app/src/main/java/de/tum/in/tumcampusapp/activities/SetupEduroamActivity.java]: This should be in `de.tum.in.tumcampusapp.auxiliary.Const`

**Comment by kordianbruck** [comment]
@jacqueline8711 I don't think its really possible. You can only try to set the config and then realise that it won't let you. 

As for your branch: in the future please directly create the feature branch in this repo, not in your fork, so its easier to check it out. Thanks! (Naming: jacqueline/<issueid>-<fancyname>)

**Pull request merged**